test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -58,7 +70,7 @@ def run_ec2_tensorflow_inference(image_uri, ec2_connection, grpc_port, region):
     docker_cmd = "nvidia-docker" if "gpu" in image_uri else "docker"
     docker_run_cmd = (
         f"{docker_cmd} run -id --name {container_name} -p {grpc_port}:8500 "
-        f"--mount type=bind,source={model_path},target=/models/mnist -e MODEL_NAME=mnist"
+        f"--mount type=bind,source={model_path},target=/models/mnist -e TEST_MODE=1 -e MODEL_NAME=mnist"
```

Comment:
Has the script been updated to accept this env change?

Reply:
The script does not needs to be changed.